compiler/rustc_data_structures/src/owned_slice.rs

```diff
@@ -4,7 +4,6 @@ use crate::sync::Lrc;
 // Use our fake Send/Sync traits when on not parallel compiler,
 // so that `OwnedSlice` only implements/requires Send/Sync
 // for parallel compiler builds.
-use crate::sync::{Send, Sync};
```

Comment:
Needs to be reverted (https://github.com/rust-lang/rust/pull/118692#issuecomment-1847908699).

Reply:
Thank you. Because of the direct revert, the unused imports check will report an error. Is it possible to distinguish this situation through the cfg macro?

Reply:
What I wanted to do in this PR is to apply trivial changes only (unconditional import removals).
False positives, allows, conditional imports can all be deferred to https://github.com/rust-lang/rust/pull/117772.
That's why I suggested reverting this removal in the comment above, and not adding a cfg.

Reply:
In this specific case explicit `sync::Send` should be used instead of an import, that's how it's done in all other places in the compiler.

Reply:
> In this specific case explicit `sync::Send` should be used instead of an import, that's how it's done in all other places in the compiler.

Got it, thank you for your patient explanation.